scripts/gen_vimdoc.py

```diff
@@ -365,6 +395,86 @@ def render_node(n, text, prefix='', indent='', width=62):
     return text
 
 
+def para_as_map(parent, indent='', width=62):
+    """Parses XML <para> as a map
+
+    The map returned may or may not contain
+    all of the following keys:
+        'text': Text in the para tag
+        'params': Information from <parameterlist>'s
+        'return': TODO
+        'seealso': TODO
+        'xrefs': TODO
```

Comment:
Not so sure on what to put for the docstrings here. Also, is `xrefs` necessary?

Reply:
not sure. xrefs might be useful for docstrings that "link" to each other, but currently it's just there "for fun".